src/components/Proposal/HeroBanner.css

```diff
@@ -0,0 +1,88 @@
+.HeroBanner {
+  position: absolute;
+  border-radius: 6px;
+  height: 100%;
+  left: 1rem;
+  right: 1rem;
+}
+
+.HeroBanner--finished {
+  border: 1px solid #e8e8e8;
+  background: linear-gradient(89deg, #f6f5f8 0%, rgba(246, 245, 248, 0) 100%);
+}
+
+.HeroBanner__Gradient {
+  background: linear-gradient(90deg, rgba(22, 20, 26, 0) 0%, #16141a 100%);
+}
+
+.HeroBanner--purple {
+  background: #a524b3;
```

Comment:
Can we add this to the theme? Can't we use purple-800?

Reply:
This is purple-A or B in figma. 

Added --fuchsia-500

<img width="465" alt="image" src="https://github.com/decentraland/governance/assets/2858950/de04e273-9430-46b4-907c-643cf67f072c">

<img width="490" alt="image" src="https://github.com/decentraland/governance/assets/2858950/69979ea0-6d11-43b7-817c-526d02295f5e">